Makefile

```diff
@@ -216,7 +216,7 @@ e2e_test: build
 # Run the code coverage tools, compute aggregate.
 unit_test_cover: build dependency_check demo
 	source build.env
-	go test $(VT_GO_PARALLEL) -count=1 -coverprofile=coverage.out ./go/...
+	go test $(VT_GO_PARALLEL) -count=1 -failfast -covermode=atomic -coverpkg=./... -coverprofile=coverage.out ./go/...
```

Comment:
What happens to the coverage if a test fails?

Reply:
The workflow fails and we run it again. That happens anyway, but the -failfast flag is just trying to short-circuit that a bit.

Reply:
got it